Backend/Backend/Controllers/PodcastController.cs

```diff
@@ -19,11 +21,13 @@ public class PodcastController : ControllerBase
 
     private readonly IPodcastService _podcastService;
     private readonly IAuthService _authService;
+    private readonly AppDbContext _db;
 
-    public PodcastController(IPodcastService podcastService, IAuthService authService)
+    public PodcastController(IPodcastService podcastService, IAuthService authService, AppDbContext db)
```

Comment:
Expanding on Matts comments, if we arent doing db operations in the controller, we dont need to include it in the constructor